frontend/src/components/organisms/ManageAttendees.tsx

```diff
@@ -72,10 +74,36 @@ type FormValues = {
   endTime: Date;
 };
 
+interface ViewCancelMessageModalBodyProps {
+  handleClose: () => void;
+  eventData?: any;
+  userid?: string;
+}
+
+const ViewCancelMessageModalBody = ({
+  handleClose,
+  eventData,
+  userid,
+}: ViewCancelMessageModalBodyProps) => {
+  const eventAttendance = eventData?.attendees?.find(
+    (attendee: any) => attendee.userId === userid
+  );
```

Comment:
If this is all we need the eventData for, perhaps we could just make a request that gives us the status of the user from the server?

Reply:
Yep, made a new endpoint (in the other branch) called `GET /events/:eventid/attendees/:userid`